packages/frontend/editor-ui/src/views/WorkflowsView.vue

```diff
@@ -974,7 +1007,95 @@ const deleteFolder = async (folderId: string, workflowCount: number, subFolderCo
 			title: i18n.baseText('folders.delete.success.message'),
 			type: 'success',
 		});
-		await onFolderDeleted({ folderId });
+		await onFolderDeleted({ folderId, workflowCount, folderCount: subFolderCount });
+	}
+};
+
+const moveFolder = async (payload: {
+	folder: { id: string; name: string };
+	newParent: { id: string; name: string };
+}) => {
+	if (!route.params.projectId) return;
+	try {
+		await foldersStore.moveFolder(
+			route.params.projectId as string,
+			payload.folder.id,
+			payload.newParent.id,
+		);
+		const isCurrentFolder = currentFolderId.value === payload.folder.id;
+		const newFolderURL = `/projects/${route.params.projectId}/folders/${payload.newParent.id}/workflows`;
```

Comment:
We should use `router.resolve` here, so that if the route is updated, this is automatically updated.